test/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.1)
+
+set(CMAKE_CXX_STANDARD 98)
+
+link_libraries(certificate_verifier gtest_main ${OPENSSL_LIBRARIES})
+include_directories(${OPENSSL_INCLUDE_DIR})
+
+set(CRL_SET_OBJ "AGCertificateVerifierTestCrlSet.o")
+set(CRL_SET_BIN "AGCertificateVerifierTestCrlSet.bin")
+add_library(test_verifier_crl_set_blob STATIC "${CMAKE_CURRENT_BINARY_DIR}/${CRL_SET_OBJ}")
+set_source_files_properties("${CMAKE_CURRENT_BINARY_DIR}/${CRL_SET_OBJ}" PROPERTIES EXTERNAL_OBJECT true GENERATED true)
+set_target_properties(test_verifier_crl_set_blob PROPERTIES LINKER_LANGUAGE C)
+add_custom_command(OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/${CRL_SET_OBJ}"
+        PRE_BUILD
+        COMMAND wget -nv -O "${CRL_SET_BIN}" "http://clients2.google.com/service/update2/crx?response=redirect\\&x=id%3Dhfnkpimlhhgieaddgfemjhofmfblmnib%26v=%26uc"
```

Comment:
On my system, wget is not installed by default. `cmake --build .` fails:

```
[ 80%] Generating AGCertificateVerifierTestCrlSet.o
/bin/sh: wget: command not found
make[2]: *** [test/AGCertificateVerifierTestCrlSet.o] Error 127
```

Add a check to CMakeLists.txt: `wget` binary should be in the `PATH`.